spec/models/project_spec.rb

```diff
@@ -33,7 +33,7 @@
       expect(subject).to_not be_valid
     end
 
-    it "should not accept invalid Pivotal Tracker URL" do
+    it 'should not accept invalid Pivotal Tracker URL' do
```

Comment:
@tansaku note that the link tested here has an extra level before the `project_id` which makes it invalid. A valid link should be of this format
```
https://www.pivotaltracker.com/s/../912312
```

and not

```
https://www.pivotaltracker.com/s/../../912312
````
It's not really about it having the s.